test/integration/css/test/css-modules.test.js

```diff
@@ -427,19 +427,22 @@ describe('CSS Modules Composes Ordering', () => {
     })
   }
 
-  describe('Development Mode', () => {
-    beforeAll(async () => {
-      await remove(join(appDir, '.next'))
-    })
-    beforeAll(async () => {
-      appPort = await findPort()
-      app = await launchApp(appDir, appPort)
-    })
-    afterAll(async () => {
-      await killApp(app)
-    })
+  // TODO: re-enable this once fixed on turbopack side
+  ;(process.env.TURBOPACK ? describe.skip : describe)('production mode', () => {
```

Comment:
This looks wrong, it shouldn't be called `production mode`.

Reply:
ah yeah should be "dev mode"